src/Octofet.cpp

```diff
@@ -0,0 +1,36 @@
+#include "octofet.h"
```

Comment:
```suggestion
#include "Octofet.h"
```

Linux and macOS are case-sensitive. Please, verify the library compiles correctly on Linux.

Reply:
Done. The library compiles and works correctly on Linux.